presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java

```diff
@@ -66,6 +66,10 @@ private void setupLdap(LdapServerConfig ldapServerConfig, Binder binder)
             checkState(ldapServerConfig.getUserObjectClass() != null, "Missing property 'authentication.ldap.user-object-class'");
         }
 
+        if (ldapServerConfig.getLdapUrl().startsWith("ldap://")) {
```

Comment:
Add a `@Pattern` validation on the config that checks for `ldaps://`

Example config and tests:
https://github.com/airlift/airlift/blob/master/node/src/main/java/io/airlift/node/NodeConfig.java
https://github.com/airlift/airlift/blob/master/node/src/test/java/io/airlift/node/TestNodeConfig.java